scripts/teamDataScript.js

```diff
@@ -213,11 +215,9 @@ function savePitAnswers() //onclick for save pit button
                 areAnyFieldsBlank = true;
             }
         }
-        var pair = {};
-        pair[question] = answer;
-        if (JSON.stringify(pair) != '{}') //first row reads {} as its reading the table header
+        if (question != 'Question') //first row reads 'Question' as its a table header
         {
-            all_team_QA_pairs_obj.push(pair);
+            all_team_QA_pairs_obj[question] = answer;
         }
```

Comment:
flip this if around, if it's the first row, leave early - otherwise handle it:

```js
if (question == 'Question') {
   return;
}

all_team_QA_pairs_obj[question] = answer;
```